src/PlusCommon/PlusCommon.h

```diff
@@ -47,13 +47,13 @@ enum PlusImagingMode
 
 /* Define case insensitive string compare for Windows. */
 #if defined( _WIN32 ) && !defined(__CYGWIN__)
-#  if defined(__BORLANDC__)
-#    define STRCASECMP stricmp
-#  else
-#    define STRCASECMP _stricmp
-#  endif
+  #if defined(__BORLANDC__)
+    #define STRCASECMP stricmp

```

Comment:
As we move towards std::string, we should probably retire STRCASECMP.
It is mostly used for XML attribute value checks, such as 

`if (toolDataElement->GetAttribute("Type") != NULL && STRCASECMP(toolDataElement->GetAttribute("Type"), "Tool") != 0)`

If we add a helper method for that then the code will be more compact and we can control at one place how this comparison is implemented.

For the rest of the STRCASECMP we need to check, but I guess we should be able to replace them with the new method.